### PR TITLE
homework 3

### PR DESCRIPTION
**Pull request by ilyabaranowski**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,194 @@
...
+            <br />
+            <input type="text" required id="Name" name="FName" placeholder="Name" />
+            <br />
+            <label for="Email">Email</label>
+            <br />
+            <input type="text" required id="Email" name="Email" />
+            <br />
+            <label for="Passwod">Password</label>
+            <br />
+            <input type="password" required id="Passwod" name="Passwod" />
```
Comment (ghost) [inline index.html:146]: Should be required.

File: index.html [hunk 2]
```diff
@@ -1,13 +1,194 @@
...
+    <fieldset>
+        <legend>My first form</legend>
+        <form action="" method="">
+            <label for="Name">Name</label>
+            <br />
+            <input type="text" required id="Name" name="FName" placeholder="Name" />
+            <br />
+            <label for="Email">Email</label>
+            <br />
+            <input type="text" required id="Email" name="Email" />
```
Comment (ghost) [inline index.html:142]: Should be required.

File: index.html [hunk 3]
```diff
@@ -1,13 +1,194 @@
...
+            <td>Groups the footer content in a table</td>
+        </tr>
+    </table>
+    <br />
+    <fieldset>
+        <legend>My first form</legend>
+        <form action="" method="">
+            <label for="Name">Name</label>
+            <br />
+            <input type="text" required id="Name" name="FName" placeholder="Name" />
```
Comment (ghost) [inline index.html:138]: Should be required.

File: index.html [hunk 4]
```diff
@@ -1,13 +1,194 @@
...
+            <label for="Hyper Text Transmission Protocol">Hyper Text Transmission Protocol</label>
+            <br />
+            <input id="Hyper Text Transfer Protocol" type="radio" name="What does HTTP stand for?"
+                value="Hyper Text Transfer Protocol" />
+            <label for="Hyper Text Transfer Protocol">Hyper Text Transfer Protocol</label>
+            <h5>Comment</h5>
+            <textarea name="Comment" cols="30" rows="5" style="resize: yes;" placeholder="Comment"
+                maxlength="500"></textarea>
+            <h5>What does HTML stand for?</h5>
+            <select name="What does HTML stand for?">
```
Comment (ghost) [inline index.html]: Attr name should contain a short key like `answer`.

File: index.html [hunk 5]
```diff
@@ -1,13 +1,194 @@
...
+                <option value="Your answer">Your answer</option>
+                <option value="Hyper links and Text Markup Language">Hyper links and Text Markup Language</option>
+                <option value="Home Tool Markup Language">Home Tool Markup Language</option>
+                <option value="Hyper Text Markup Language">Hyper Text Markup Language</option>
```
Comment (ghost) [inline index.html]: Attr value should contain shot value, like `answer1`.

File: index.html [hunk 6]
```diff
@@ -1,13 +1,194 @@
...
+            </select>
+            <p><input type="submit" /></p>
```
Comment (ghost) [inline index.html]: Remove `p` tag.

File: index.html [hunk 7]
```diff
@@ -1,13 +1,194 @@
...
+        odio non dignissim auctor,
+        arcu augue mattis orci,
+        quis fringilla nulla mi ac erat. Ut quis metus eget metus auctor elementum tincidunt at tortor. Quisque eros
+        massa,
+        eleifend non tempor non,
+        porttitor at risus. Curabitur laoreet nulla neque. Nulla interdum elementum felis quis egestas. Curabitur et
+        tristique massa.&#128213;
+    </p>
+    <br />
+    <p><img src="https://hungarytoday.hu/wp-content/uploads/2018/02/18ps27.jpg" alt="Hide the Pain Harold" /></p>
```
Comment (ghost) [inline index.html]: Remove `p` tag.

File: index.html [hunk 8]
```diff
@@ -1,13 +1,194 @@
...
-    </head>
-    <body>
-        <h1>My first page</h1>
-        <p><a href="./about/index.html"></a></p>  
-    </body>
-    </html>
\ No newline at end of file
+<!DOCTYPE html />
+<html lang="en">
+
```
Comment (ghost) [inline index.html]: Remove empty line.

File: index.html [hunk 9]
```diff
@@ -1,13 +1,194 @@
...
+        table,
+        th,
+        td {
+            width: 900px;
+            background: white;
+            border: 1px solid;
+        }
+    </style>
+</head>
+
```
Comment (ghost) [inline index.html]: Remove empty line.

File: index.html [hunk 10]
```diff
@@ -1,13 +1,194 @@
...
+                <option value="Your answer">Your answer</option>
+                <option value="Hyper links and Text Markup Language">Hyper links and Text Markup Language</option>
+                <option value="Home Tool Markup Language">Home Tool Markup Language</option>
+                <option value="Hyper Text Markup Language">Hyper Text Markup Language</option>
+            </select>
+            <p><input type="submit" /></p>
+        </form>
+    </fieldset>
+</body>
+
```
Comment (ghost) [inline index.html]: Remove empty line.

File: index.html (hunk 4 above)
Comment (ilyabaranowski) [reply index.html]: changed to answer

File: index.html (hunk 5 above)
Comment (ilyabaranowski) [reply index.html]: chaged

File: index.html (hunk 6 above)
Comment (ilyabaranowski) [reply index.html]: removed

File: index.html (hunk 7 above)
Comment (ilyabaranowski) [reply index.html]: removed

File: index.html (hunk 8 above)
Comment (ilyabaranowski) [reply index.html]: removed

File: index.html (hunk 9 above)
Comment (ilyabaranowski) [reply index.html]: removed

File: index.html (hunk 10 above)
Comment (ilyabaranowski) [reply index.html]: removed

**Pull request merged**